crates/autopilot/src/arguments.rs

```diff
@@ -479,6 +493,24 @@ impl FromStr for FeePolicy {
     }
 }
 
+#[derive(clap::Parser, Clone)]
+pub struct CircuitBreaker {
+    pub circuit_breaker_authenticator_pk: Option<H256>,
+    pub circuit_breaker_solvers: Option<Vec<H160>>,
+}
+
+impl Debug for CircuitBreaker {
+    fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
```

Comment:
This can be avoided with `#[derivative(Debug = "ignore")]`

Reply:
Very cool, I didn't know that, thanks a lot @squadgazzz !